src/models/Resource.ts

```
@@ -0,0 +1,30 @@
+import { Schema, model, Document } from 'mongoose';
```

ESLint

[error] Replace `'mongoose'` with `"mongoose"`

(prettier/prettier)